projects/super_res/super_res.py

```diff
@@ -0,0 +1,203 @@
+import xarray as xr
+import numpy as np
+import fv3viz
+from vcm.catalog import catalog
+import matplotlib.pyplot as plt
+import os
+import cartopy.crs as ccrs
+
+GRID = catalog["grid/c384"].read()
+
+DPI = 800
+
+
+def repeat_arr(arr, upscale_factor):
+    # This repeats values along the last dimension
+    return np.repeat(arr, upscale_factor, axis=-1)
+
+
+def upscale_nearest(ds, upscale_factor, dim):
```

Comment:
nit: Not sure if we are actually going to check the types of these arguments, but it is helpful to the reader to know, for example that upscale_factor is an int.

Reply:
nit: I would find it helpful to include "neighbor" in this function name.